cypress/integration/html-reader/toc-navigation.ts

```diff
@@ -66,7 +66,7 @@ describe('navigating an EPUB page', () => {
     cy.findByRole('button', { name: 'Table of Contents' }).click();
 
     cy.log('open chapter 1');
-    cy.findByRole('menuitem', { name: 'I: Down the Rab­bit-Hole' }).click();
+    cy.findByRole('listitem', { name: 'I: Down the Rab­bit-Hole' }).click();
```

Comment:
I'm going to be annoying and ask about this stuff now - was `menuitem` being flaky?

Reply:
No I changed the html to render list items instead of menuitems. I think that's more semantic anyways for this use case